awscli/compat.py

```diff
@@ -48,7 +55,25 @@ def __enter__(self):
     def __exit__(self, type, value, traceback):
         if sys.platform == "win32":
             import msvcrt
-            msvcrt.setmode(sys.stdout.fileno(), self.previous_mode)                
+            msvcrt.setmode(sys.stdout.fileno(), self.previous_mode)
+
+
+def bytes_print(statement, stdout=None):
+    """
+    This function is used to write raw bytes to stdout.
+    """
+    if stdout is None:
+        stdout = sys.stdout
+
+    if PY3:
```

Comment:
In the future, please try to avoid `PY3` conditionals within a function like this.  The prevailing pattern in this module is to separate python2 vs. python3 code so there's specific versions of a function based on python version.  Checking whether or not we're in python3 is not something that needs to happen every time the function is called, it only needs to happen once.